pages/index.js

```diff
@@ -1,12 +1,17 @@
-import SpotList from "../components/SpotList.js";
+import Spots from "../pages/spots/index.js";
 import Footer from "../components/Footer.js";
 import Header from "../components/Header.js";
+import { useState } from "react";
+import { mockSpots } from "../helpers/mockSpots.js";
+
 export default function Home() {
+  const [spots, setSpots] = useState(mockSpots);
   return (
-    <div>
+    <>
       <Header />
-      <SpotList />
+      <Spots spots={spots} setSpots={setSpots} />
```

Comment:
`<Spots>` in itself is a page currently (because it exists within the pages folder).

I do not think that is what you want.

Move `pages/spots/index.js` to `components/Spots/index.js` so that it becomes a normal component.

You'd then probably don't want to pass the whole `setSpots` function but only your `toggleBookmark` function that you have created in the _app.js.

```suggestion
      <Spots spots={spots} toggleBookmark={toggleBookmark} />
```